### PR TITLE
Updating and correcting the new documentation

### PR DESCRIPTION
**Pull request by GLS-SSV**
PR for ticket #440

### CODE REVIEW

File: Doc/Orbiter User Manual/LAUNCHPAD.tex [hunk 1]
```diff
@@ -48,72 +48,37 @@ \subsection{Scenarios tab}
 The Tutorials folder contains pre-recorded flights with on-screen annotations that explain different aspects and stages of spaceflight missions.
```
Comment (n7275) [inline Doc/Orbiter User Manual/LAUNCHPAD.tex:48]: \item

File: Doc/Orbiter User Manual/LAUNCHPAD.tex [hunk 2]
```diff
@@ -172,61 +137,336 @@ \subsection{Visual effects tab}
...
 
+\subsubsection{Physics settings}
+\begin{figure}[H]
+	\centering
+	\subfigure{\includegraphics[width=0.25\textwidth]{launchpad_physics.png}}
+\end{figure}
+
+\textbf{Perturbations}
+\begin{itemize}
+\item \textbf{Nonspherical gravity sources:} This option activates a more complex gravity force calculation which can take into account perturbations in the gravitational potential due to non spherical celestial body shapes, thus allowing more accurate orbit predictions. Note that this option can make orbital calculations more difficult, and may reduce the reliability of instruments that don't take this effect into account. For a planet to make use of the perturbation code, its configuration file must supply shape parameters via the \textit{JCoeff} entry. For background and technical implementation details please refer to "Nonspherical gravitational field perturbations" in Orbiter Technical Reference.
```
Comment (n7275) [inline Doc/Orbiter User Manual/LAUNCHPAD.tex:179]: "must supply shape parameters via the JCoeff entry"^, or provide a Spherical Harmonic model with the _GravModelPath_ and _GravCoeffCutoff_ entries

File: Doc/Orbiter User Manual/LAUNCHPAD.tex (hunk 1 above)
Comment (GLS-SSV) [reply Doc/Orbiter User Manual/LAUNCHPAD.tex:48]: Fixed, thanks!

File: Doc/Orbiter User Manual/LAUNCHPAD.tex (hunk 2 above)
Comment (GLS-SSV) [reply Doc/Orbiter User Manual/LAUNCHPAD.tex:179]: Fixed, thanks!

**Review by n7275 — APPROVED** [review]

**Pull request merged**